airbyte-integrations/connectors/source-mixpanel/source_mixpanel/source.py

```diff
@@ -79,10 +79,9 @@ def check_connection(self, logger: AirbyteLogger, config: Mapping[str, Any]) ->
         try:
             config = self._validate_and_transform(config)
             auth = self.get_authenticator(config)
-            FunnelsList.max_retries = 0
-            funnels = FunnelsList(authenticator=auth, **config)
-            funnels.reqs_per_hour_limit = 0
-            next(read_full_refresh(funnels), None)
+            annotations = Annotations(authenticator=auth, **config)
```

Comment:
noting that we do NOT know that using the `Annotations` stream will help users with a basic account. we only know `funnels` doesn't work anymore

Reply:
as a temporary solution, we can make a check on all streams and stop at the first one where we will get success